Base/usr/share/man/man5/ini.md

```diff
@@ -0,0 +1,21 @@
+## Name
+
+INI - generic config file format (.ini)
+
+## Description
+
+INI files serve as human-readable configuration files.
+
+INI files consist of key-value configuration data pairs separated by a unique group in square brackets.
+
+Additionally, [`Userland/Libraries/LibCore/ConfigFile.cpp`](../../../../../Userland/Libraries/LibCore/ConfigFile.cpp)
+supports comments: '#' and ';' will skip entire line (as '//' in C++).
```

Comment:
This is not complete enough; AFAIK comments are only detected when the line starts with either character and not if it includes them later on.

Reply:
Indeed, the logic you described is correct.